python/doc/examples/probabilistic_modeling/random_vectors/plot_ratio_of_uniforms_exploitation.py

```diff
@@ -0,0 +1,455 @@
+    boundary_1_fct = ot.SymbolicFunction(
+        "x", "(" + str(vSup) + "/x)^(1/" + str(r) + ")"
+    )
+    if vSup / (uSup**r) < xSup:
+        boundary_1 = boundary_1_fct.draw(vSup / (uSup**r), xSup, 101)
+        boundary_1.setColors(["blue"])
+        leg.add(r"$\varphi(\mathcal{F}_1)$")
+        g.add(boundary_1)
+    if vInf / (uSup**r) < uSup:
+        boundary_2_fct = ot.SymbolicFunction(
+            "x", "(" + str(vInf) + "/x)^(1/" + str(r) + ")"
+        )
+        boundary_2 = boundary_2_fct.draw(xInf, vInf / (uSup**r), 101)
+        boundary_2.setColors(["red"])
+        leg.add(r"$\varphi(\mathcal{F}_2)$")
+        g.add(boundary_2)
+    boundary_3 = ot.Curve([[vInf / (uSup**r), uSup], [vSup / (uSup**r), uSup]])
+    boundary_3.setColor("green")
+    g.add(boundary_3)
+    leg.add(r"$\varphi(\mathcal{F}_3)$")
+    g.setLegends(leg)
+
+    return g
+
+
+# %%
+# Illustration.
+# -------------
+# We create the function :math:`\log f` defined by:
+#
+# .. math::
+#    :label: robertEx
+#
+#    \log f(x) = \log(\cos(x)) + x,  \quad  x \in \left[-\dfrac{\pi}{2}, \dfrac{\pi}{2}\right]
+#
+# and the :class:`~openturns.experimental.RatioOfUniforms`.
+
+
+log_UnscaledPDF = ot.SymbolicFunction("x", "log(cos(x)) + x")
+range_PDF = ot.Interval(-pi / 2.0, pi / 2.0)
+ratio_Algo = otexp.RatioOfUniforms(log_UnscaledPDF, range_PDF)
+
+# %%
+# We draw the boundary :math:`\partial A_{f,r}` and the bounding box :math:`\partial \tilde{A}_{f,r}`
+# in the :math:`(u,v)`-space.
+g_UV = draw_RatioOfUniforms_UV(ratio_Algo)
+g_UV.setTitle(
+    rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+)
+
+# %%
+# We draw the boundary :math:`\partial \varphi(A_{f,r})` and the bounding box :math:`\partial \varphi(\tilde{A}_{f,r})`
+# in the :math:`(x,y)`-space.
+# sphinx_gallery_thumbnail_number = 1
+g_XY = draw_RatioOfUniforms_XY(ratio_Algo, 1001)
+g_XY.setTitle(
+    rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+)
+
+grid_default = ot.GridLayout(1, 2)
+grid_default.setGraph(0, 0, g_UV)
+grid_default.setGraph(0, 1, g_XY)
+grid_default.setTitle(r"$\log f(x) = \log(\cos(x)) + x, x \in ]-\pi/2, \pi/2[$")
+view = otv.View(grid_default)
+
+# %%
+# We change the :math:`r` value and we draw the same plots.
+grid_full = ot.GridLayout(4, 2)
+for i, r in enumerate([0.5, 0.8, 1.2, 2.0]):
+    ratio_Algo.setR(r)
+    g_UV = draw_RatioOfUniforms_UV(ratio_Algo)
+    g_UV.setLegendPosition("")
+    g_UV.setTitle(
+        rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+    )
+
+    g_XY = draw_RatioOfUniforms_XY(ratio_Algo, 1001)
+    g_XY.setLegendPosition("")
+    g_XY.setTitle(
+        rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+    )
+
+    grid_full.setGraph(i, 0, g_UV)
+    grid_full.setGraph(i, 1, g_XY)
+
+grid_full.setTitle(r"$\log f(x) = \log(\cos(x)) + x,  \quad x \in ]-\pi/2, \pi/2[$")
+view = otv.View(grid_full)
+
```

Comment:
maybe add a sentence of analysis based on the plots on the influence of the parameter r on the shape of the boundaries to help the reader to understand the role of this parameter ?

Reply:
yes... not easy to understand the utility of this parameter... I added a comment....